packages/smooth_app/lib/background/background_task_image.dart

```diff
@@ -238,6 +245,11 @@ class BackgroundTaskImage extends AbstractBackgroundTask {
     } catch (e) {
       // not likely, but let's not spoil the task for that either.
     }
+    try {
+      File(_getCroppedPath()).deleteSync();
+    } catch (e) {
+      // possible, but let's not spoil the task for that either.
```

Comment:
@M123-dev For this particular case (`_getCroppedPath()`) that wouldn't be such a good idea as sometimes - when the crop is actually the "full image" - we don't create this path/File. We would then have to handle a special flag for that.
Regarding the other cases, I wouldn't say it would be completely pointless, but almost: if it fails the rest of the app will work OK, including the next uploads.
And there's already an open issue regarding background task init+garbage collecting, that would get rid of those files anyway.
One day we may log this, but today that looks like a "just in case" that we don't really need.
Anyway sentry is overcrowded with camera issues that make it useless.